packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -69,15 +71,65 @@ const confirmDialogsPropTypes = {
  * ConfirmDialogPublishAll
  * -----------------------------------------------------------------------------------------------*/
 
-const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+const ConfirmDialogPublishAll = ({
+  isOpen,
+  onToggleDialog,
+  isConfirmButtonLoading,
+  onConfirm,
+  slug,
+}) => {
   const { formatMessage } = useIntl();
+  const { get } = useFetchClient();
+  const { selectedEntries } = useTableContext();
+
+  const { data: countDraftRelations } = useQuery(
+    ['content-manager', 'draft-relations'],
+    async () => {
+      try {
+        const {
+          data: { data },
+        } = await get(
+          `/content-manager/collection-types/${slug}/actions/countManyEntriesDraftRelations?${stringify(
+            { ids: selectedEntries },
+            { encode: false }
+          )}`
+        );
+
+        return data;
+      } catch (err) {
+        console.error(err.message);
+
+        return null;
```

Comment:
Stray console.log. You don't need to return here.

Reply:
I added the return because otherwise I have a warning in the editor 
<img width="780" alt="Schermata 2023-07-05 alle 14 51 09" src="https://github.com/strapi/strapi/assets/2589748/307833d2-8bf5-4c6b-bfe5-7ec00da98363">

Reply:
which is why you don't typically use `try/catch` in `react-query` because it already has error handling – https://tanstack.com/query/v3/docs/react/guides/query-functions

Reply:
done, thanks